sdk/nodejs/runtime/closure/parseFunction.ts

```diff
@@ -424,14 +424,14 @@ function computeCapturedVariableNames(file: ts.SourceFile): CapturedVariables {
     const result: CapturedVariables = { required: new Map(), optional: new Map() };
 
     for (const key of required.keys()) {
-        if (required.has(key) && !isBuiltIn(key)) {
```

Comment:
the 'has' was a holdover from when we weren't using maps here and were just using plain objects.  Now that it's an actual map, 'has' is pointless since we're enumerating over the actual keys of the map.